hooks/post-command

```diff
@@ -72,14 +72,26 @@ get_codecov_uploader() {
   fi
 
   # One-time step
-  curl \
-      -fSs \
-      --retry 5 \
-      https://keybase.io/codecovsecurity/pgp_keys.asc \
+  if [[ "${BUILDKITE_PLUGIN_CODECOV_PGP_PUBLIC_KEY_URL-"false"}" = "" ]]; then
+    debug "Importing bundled Codecov's PGP public key : $(pwd)/pgp_keys.asc" 
+    cat $(pwd)/pgp_keys.asc \
```

Comment:
```suggestion
    cat "$(pwd)/pgp_keys.asc" \
```

due to https://www.shellcheck.net/wiki/SC2046

Reply:
Fixed in https://github.com/joscha/codecov-buildkite-plugin/pull/28/commits/e62af2b9ad26c4851b5964adf96417dd74f2e0a8